src/Shared/runtime/Quic/Implementations/MsQuic/Internal/MsQuicApi.cs

```diff
@@ -143,6 +143,12 @@ static MsQuicApi()
 
             OperatingSystem ver = Environment.OSVersion;
 
+            if (ver.Platform == PlatformID.Win32NT && ver.Version < new Version(10, 0, 19041, 0))
```

Comment:
Justin, I didn't realize you'd removed this in aspnet. This PR won't work then. I'll keep it long enough to figure out the CLA, but go ahead and merge your other PR.